### PR TITLE
Remove already triggered migrations

### PR DESCRIPTION
**Pull request by expenses**

### CODE REVIEW

File: runtime/kusama/src/lib.rs
Comment (kianenigma) [inline runtime/kusama/src/lib.rs:1067]: Can you remove this

File: runtime/polkadot/src/lib.rs
Comment (kianenigma) [inline runtime/polkadot/src/lib.rs:1062]: and this as well 😂  all of these are already in v30 branch.

File: runtime/kusama/src/lib.rs
Comment (expenses) [reply runtime/kusama/src/lib.rs:1067]: Remove `KillOffchainPhragmenStorageTest`?

**Comment by kianenigma** [comment]
from v30 

https://github.com/paritytech/polkadot/blob/v0.8.30-rc1/runtime/polkadot/src/lib.rs#L1068
https://github.com/paritytech/polkadot/blob/v0.8.30-rc1/runtime/kusama/src/lib.rs#L1076

@s3krit if we need rc2, we backport some changes right? these will stay in the release anyway?

File: runtime/kusama/src/lib.rs
Comment (kianenigma) [reply runtime/kusama/src/lib.rs:1067]: Yes

**Review by kianenigma — APPROVED** [review]
Should remove the definition of `FixPolkadotCouncilVotersDeposit` and `KillOffchainPhragmenStorageTest`  as well, as they don't serve any purpose any more.

**Comment by expenses** [comment]
Probably best to wait for another review so I don't merge this and break things.

**Comment by s3krit** [comment]
> @s3krit if we need rc2, we backport some changes right? these will stay in the release anyway?

We do have an rc2 now ;) But yes, only relevant changes get backported so this is fine.

**Comment by expenses** [comment]
What should I do with this? Merge when ready?

**Comment by kianenigma** [comment]
Yes.

**Comment by expenses** [comment]
Okedoke

**Comment by expenses** [comment]
bot merge

**Comment by expenses** [comment]
bot merge

**Comment by expenses** [comment]
Okay whatever

**Pull request merged**
